notebooks/workflow_cost_estimator.py

```diff
@@ -101,18 +92,14 @@ def estimate_job_cost(cpus: int, memory_gb: int, runtime_hours: float, preemptib
                       (8, 32, 10, False),
                       (10, 64, 5, True),
                       (8, 32, 10, True)]
-    print("%8s" % "cpus",
-          "%8s" % "memory",
-          "%8s" % "runtime",
-          "%12s" % "preemptible",
-          "%8s" % "cost")
+    report = pd.DataFrame()
     for cpus, memory_gb, runtime_hours, preemptible in configurations:
         cost = estimate_job_cost(cpus, memory_gb, runtime_hours, preemptible)
-        print("%8i" % cpus,
-              "%6iGB" % memory_gb,
-              "%7ih" % runtime_hours,
-              "%12s" % str(preemptible),
-              "%8s" % ("$%.2f" % cost))
+        report = report.append(dict(cost=cost, cpus=cpus, memory=memory_gb, duration=runtime_hours, preemptible=preemptible), ignore_index=True)
+    report['preemptible'] = report['preemptible'].astype('bool')
+
+with callysto.Cell("python"):
+    report.style.format(dict(cost="${:.2f}", duration="{:.2f}h", memory="{:.0f}GB"))
```

Comment:
Why are CPUs and preemptibles not reported anymore, but memory is?

Reply:
All fields are reported.

It may be instructive to install the notebook into a test workspace and check output. The workspace should contain finished workflows.